sh.fhs.KatawaShoujoReEngineered.yaml

```diff
@@ -0,0 +1,106 @@
+      - mv "${FLATPAK_DEST}/include/python3.11/pygame_sdl2" "${FLATPAK_DEST}/include/pygame_sdl2"
+    sources:
+      - type: git
+        url: https://github.com/renpy/pygame_sdl2.git
+        tag: renpy-8.1.0.23050902
+
+  - name: renpy
+    buildsystem: simple
+    build-options:
+      arch:
+        aarch64:
+          env:
+            RENPY_DEPS_INSTALL: "/app::/usr::/lib::/lib64::/lib/aarch64-linux-gnu"
+            CFLAGS: -fcommon
+        x86_64:
+          env:
+            RENPY_DEPS_INSTALL: "/app::/usr::/lib::/lib64::/lib/x86_64-linux-gnu"
+            CFLAGS: -fcommon
+    build-commands:
+      - python module/setup.py build
+      - python module/setup.py install --root="/" --prefix="${FLATPAK_DEST}" --optimize=1
+      - install -dm755 "${FLATPAK_DEST}/share/renpy"
+      - cp -r doc launcher renpy renpy.py sdk-fonts "${FLATPAK_DEST}/share/renpy/"
+      - install -Dm755 renpy.sh "${FLATPAK_DEST}/bin/renpy"
+    sources:
+      - type: archive
+        url: https://www.renpy.org/dl/8.1.3/renpy-8.1.3-source.tar.bz2
+        sha256: e207b080acc6721eb51bdbb5f1021d18bb0b8bfb79116cf683dd9fdb5bf9cebc
+      - type: script
+        commands:
+          - python /app/share/renpy/renpy.py "$@"
+        dest-filename: renpy.sh
+
+  - name: ksre
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/ffmpeg
+      - cp -R game "${FLATPAK_DEST}/game"
+      - install -Dm755 launcher.sh "${FLATPAK_DEST}/bin/ksre"
+      - install -Dm644 -t "${FLATPAK_DEST}/share/appdata/" sh.fhs.KatawaShoujoReEngineered.appdata.xml
+      - install -Dm644 -t "${FLATPAK_DEST}/share/applications/" sh.fhs.KatawaShoujoReEngineered.desktop
+      - install -Dm644 48x48.png "${FLATPAK_DEST}/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png"
+      - install -Dm644 64x64.png "${FLATPAK_DEST}/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png"
+      - install -Dm644 128x128.png "${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png"
+      - install -Dm644 256x256.png "${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png"
+      - install -Dm644 512x512.png "${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png"
+      - sed -i 's/config.developer = "auto"/config.developer = False/g' /app/game/config.rpy
+      - renpy /app/game compile
+    sources:
+      - type: git
+        url: https://codeberg.org/fhs/katawa-shoujo-re-engineered.git
+        tag: v1.3.0
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+      - type: script
+        dest-filename: launcher.sh
+        commands:
+          - renpy /app/game "$@"
+      - type: file
+        path: sh.fhs.KatawaShoujoReEngineered.appdata.xml
+      - type: file
+        path: sh.fhs.KatawaShoujoReEngineered.desktop
+      - type: file
+        path: icons/48x48.png
+      - type: file
+        path: icons/64x64.png
+      - type: file
+        path: icons/128x128.png
+      - type: file
+        path: icons/256x256.png
```

Comment:
remove all of these and only keep the biggest one since they don't come from the upstream source, which ideally they should.

Reply:
fixed by obtaining svg icon from upstream